<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="ownerField">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>421</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="opponentField">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>421</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="makeTurnButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сделать ход</string>
     </property>
    </widget>
    <widget class="QPushButton" name="placeShipButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Поставить корабль</string>
     </property>
    </widget>
    <widget class="QLabel" name="currentGameStatelabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>320</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Текущее состояние игры</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="weaponsBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>330</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Орудия</string>
     </property>
     <widget class="QComboBox" name="weaponsComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="getWeaponInfoButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Подробно</string>
      </property>
     </widget>
     <widget class="QLabel" name="weaponUsesLeftLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>66</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="activateWeaponCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Использовать</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="autoPlaceShipsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Разместить корабли автоматически</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:black</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: red;</string>
      </property>
      <property name="text">
       <string>×</string>
      </property>
     </widget>
     <widget class="QToolButton" name="hideButton">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: lightblue;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="yourTurnTimeRemainLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="opponentWaitingRemainingTimeLabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="shipsAmountLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
